query-node/mappings/content-directory/entity/index.ts

```diff
@@ -194,19 +184,10 @@ async function contentDirectory_EntityRemoved(db: DB, event: SubstrateEvent): Pr
   const entityId = decode.stringIfyEntityId(event)
   const where: IWhereCond = { where: { id: entityId } }
 
-  const classEntity = await db.get(ClassEntity, where)
-  if (classEntity === undefined) {
-    console.log(`Class not found for the EntityId: ${entityId}`)
-    return
-  }
-
-  const cls = contentDirectoryClassNamesWithId.find((c) => c.classId === classEntity.classId)
-  if (cls === undefined) {
-    console.log('Unknown class')
-    return
-  }
+  const knownClass = await getKnownClass(db, where)
+  if (!knownClass) return
```

Comment:
Isn't this just hiding the fact that the database is inconsistent with the state of the runtime?

Reply:
Its only logical that mappings have to ignore unknown classes that may have been added arbitrarily.